examples/map/track_and_coalign_region.py

```diff
@@ -0,0 +1,96 @@
+"""
+=========================================
+Tracking and Co-aligning an Active Region
+=========================================
+
+This example demonstrates how to track a particular region as a function of time,
+create a cutout around that region, and align it at each time step to get an aligned datacube.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import propagate_with_solar_surface
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###############################################################################
+# `sunpy.net.Fido` is the primary interface for searching and downloading data.
+# The following code performs a search for solar data using the Fido client from the sunpy.net module.
+query = Fido.search(a.Time('2012/3/4', '2012/3/5'),
+                    a.Instrument.aia,
+                    a.Wavelength(171*u.angstrom),
+                    a.Sample(360*u.minute))
+print(query)
+
+###############################################################################
+# Let's inspect the results and download the files.
+files = Fido.fetch(query)
+print(files)
+
+###############################################################################
+# Create a sequence of maps from the downloaded files.
+
+m_seq = sunpy.map.Map(files, sequence=True)
+
+# Plot the sequence of maps.
+
+
+fig = plt.figure(figsize=(16, 4))
+for i, m in enumerate(m_seq):
+    ax = fig.add_subplot(1, len(m_seq), i+1, projection=m)
+    m.plot(axes=ax)
+
+# The above images are the result of the query: 4 images at 6-hour intervals.
+
+###############################################################################
+# Define the region of interest using a SkyCoord.
+
+
+corner = SkyCoord(Tx=-375*u.arcsec, Ty=0*u.arcsec, frame=m_seq[0].coordinate_frame)
```

Comment:
I think this should change to a region with more structures.

Reply:
```suggestion
m_cutout = m_seq[0].submap(corner, width=500*u.arcsec, height=500*u.arcsec)
```
![Screenshot from 2024-07-19 15-15-17](https://github.com/user-attachments/assets/7d407a4a-1edd-4cf0-8736-020b8f87b300)
How about this?

Reply:
> ![Screenshot from 2024-07-19 15-15-17](https://private-user-images.githubusercontent.com/139000226/350347330-7d407a4a-1edd-4cf0-8736-020b8f87b300.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzODI3MzcsIm5iZiI6MTcyMTM4MjQzNywicGF0aCI6Ii8xMzkwMDAyMjYvMzUwMzQ3MzMwLTdkNDA3YTRhLTFlZGQtNGNmMC04NzM2LTAyMGI4Zjg3YjMwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOVQwOTQ3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZDE3MDk0YTUyNDZiYTU0YjkxMTU5MzA0YzQ5MzE1ZmU3YzMzMWYxMzAxNjdhYThiMTYxYThjYmJlYWI2YmRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.8mbPQxNxrUrTkymvvbI9MrwRHQYr5sgaYtiB6nPnWxc) How about this?
oh I guess we cannot do this here because  `WARNING: SunpyUserWarning: The conversion of these 2D helioprojective coordinates to 3D is all NaNs because off-disk coordinates need an additional assumption to be mapped to calculate distance from the observer. Consider using the context manager `SphericalScreen()`. [sunpy.coordinates.frames]
2024-07-19 15:29:29 - sunpy - WARNING: SunpyUserWarning: The conversion of these 2D helioprojective coordinates to 3D is all NaNs because off-disk coordinates need an additional assumption to be mapped to calculate distance from the observer. Consider using the context manager `SphericalScreen()`.
`
![Screenshot from 2024-07-19 15-23-43](https://github.com/user-attachments/assets/224e6de0-c6c1-481a-b356-89227378f36e)

Reply:
`m_cutout = m_seq[0].submap(corner, width=500*u.arcsec, height=500*u.arcsec)
`
![Screenshot from 2024-07-19 15-34-59](https://github.com/user-attachments/assets/e0fb7674-42c3-44f2-bc0a-1e98cf3ffaaa)
How about this?

Reply:
I liked the near limb area, we can use the context manager as well to fix the white space.